focus/operations/src/testing/integration.rs

```diff
@@ -253,3 +256,60 @@ impl Drop for RepoPairFixture {
         }
     }
 }
+
+pub fn set_extra_config_on_dense_repo(fixture: &RepoPairFixture) -> Result<()> {
+    if cfg!(not(feature = "twttr")) {
+        return Ok(());
+    }
+
+    let binary = fixture.app.git_binary();
+
+    binary
+        .command()
+        .arg("config")
+        .arg("--add")
+        .arg("--local")
+        .arg("ci.alt.remote")
+        .arg("https://git.twitter.biz/focus-test-repo-ci")
+        .current_dir(&fixture.dense_repo_path)
+        .assert()
+        .try_success()?;
+
+    binary
+        .command()
+        .arg("config")
+        .arg("--add")
+        .arg("--local")
+        .arg("--bool")
+        .arg("ci.alt.enabled")
+        .arg("true")
+        .current_dir(&fixture.dense_repo_path)
+        .assert()
+        .try_success()?;
+
+    Ok(())
+}
+
+#[cfg(feature = "twttr")]
+#[cfg(test)]
+mod twttr_test {
+    use super::RepoPairFixture;
+    use anyhow::Result;
+    use focus_testing::init_logging;
+
+    #[test]
+    fn test_ci_config_is_set_in_dense_repo() -> Result<()> {
```

Comment:
This tests that the things are set in the dense repo, why not test that they're set in the sparse repo?

Reply:
Oh, good point 👀